app-main/public/404.html

```diff
@@ -3,77 +3,12 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./css/style_404.css" />
+    <link
```

Comment:
Can you remove this please. It's only used for the text editor when submitting a new lesson.

Reply:
Updated! Sorry I missed this the first time.

Reply:
No problem!